packages/fast-development-site-react/app/components/paragraph/index.ts

```diff
@@ -0,0 +1,13 @@
+import manageJss from "@microsoft/fast-jss-manager-react";
+import { default as BaseParagraph, ParagraphStyles } from "./paragraph";
+
+/*
+ * The type returned by manageJss type is very complicated so we'll let the
+ * compiler infer the type instead of re-declaring just for the package export
+ */
+/* tslint:disable-next-line:typedef */
+const Paragraph = manageJss(ParagraphStyles)(BaseParagraph);
+type Paragraph = typeof Paragraph;
+
+export default Paragraph;
+export * from "./paragraph";
```

Comment:
Same question on the value this gives us for components that we aren't exporting and only using for testing

Reply:
See above comment.